playbooks/hadoop-build-arm64/run.yaml

```diff
@@ -0,0 +1,34 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+  tasks:
+    - name: install required packages
+      shell: |
+        set -exo pipefail
+        apt-get update
+        apt-get install -y build-essential autoconf automake libtool cmake zlib1g-dev pkg-config libssl-dev libsasl2-dev
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v2.5.0/protobuf-2.5.0.tar.bz2
+        wget -O protobuf-2.5.0-arm64.patch https://www.dropbox.com/s/713wql5cw9dfxhx/protobuf-2.5.0-arm64.patch?dl=0
+        tar xvfj protobuf-2.5.0.tar.bz2
+        cd protobuf-2.5.0
+        patch -p1 < ../protobuf-2.5.0-arm64.patch
+        ./autogen.sh
+        ./configure --prefix=/usr
+        make
+        make check
+        make install
+        protoc --version
+        cd ..
+        mkdir -p ~/.m2/repository/io/grpc/protoc-gen-grpc-java/1.15.1/
+        cd ~/.m2/repository/io/grpc/protoc-gen-grpc-java/1.15.1/
+        wget http://home.apache.org/~aajisaka/repository/io/grpc/protoc-gen-grpc-java/1.15.1/protoc-gen-grpc-java-1.15.1-linux-aarch_64.exe
+        wget http://home.apache.org/~aajisaka/repository/io/grpc/protoc-gen-grpc-java/1.15.1/protoc-gen-grpc-java-1.15.1.pom
+        wget http://home.apache.org/~aajisaka/repository/io/grpc/protoc-gen-grpc-java/1.15.1/protoc-gen-grpc-java-1.15.1.pom.sha1
+        cd /home/zuul/'{{ zuul.project.src_dir }}'
```

Comment:
{{ ansible_user_dir }}/{{ zuul.project.src_dir }}?

Reply:
Ah, yes, this seems better